tree.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable import/prefer-default-export */
+
+class Node {
+    constructor(data) {
+      this.data = data;
+      this.left = null;
+      this.right = null;
+    }
+  }
+  export class Tree {
+    constructor() {
+      this.root = null;
+    }
+    
+    /**
+     * Initializes the binary search tree by setting the root node to null.
+     *
+     * @returns {null} - Always returns null.
+     */
+
+    createTree() {

```

Comment:
this is not needed, we should use constructor only, `new Tree()`

Reply:
Implemented the changes.